src/app/organization/organization-routing.module.ts

```diff
@@ -322,11 +324,29 @@ const routes: Routes = [
         },
         {
           path: 'holidays',
-          component: HolidaysComponent,
           data: { title: extract('Manage Holidays'), breadcrumb: 'Manage Holidays' },
-          resolve: {
-            offices: OfficesResolver
-          }
+          children: [
+            {
+              path: '',
+              component: HolidaysComponent,
+              resolve: {
+                offices: OfficesResolver
+              }
+            },
+            {
+              path: ':holidayId',
+              data: { title: extract('View Holidays'), routeParamBreadcrumb: 'holidayId' },
+              children: [
+                {
+                  path: '',
+                  component: ViewHolidaysComponent,
+                  resolve: {
+                    holidays: HolidayResolver
+                  }
+                }
+              ]
+            }
+          ]
```

Comment:
Please refer to the routing of other components. This can be reduced well.

Reply:
refactored. Please check